### PR TITLE
Completed css-four-column-layout. | Seungjoo Kim

### PR DESCRIPTION
**Pull request by SeungjooKimSJ**
image

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  text-align: center;
+
+  /* border: 1px solid red; */
+  position: relative;
+  width: 15%;
+  margin: 0 0.5rem;
+}
+
+.screen-img {
+  /* border: 1px solid black; */
```
Comment (thebearingedge) [inline css-four-column-layout/styles.css:35]: Please delete all commented code.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  position: relative;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:27]: This line is unnecessary

File: css-four-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  width: 15%;
+  margin: 0 0.5rem;
+}
+
+.screen-img {
+  position: relative;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:33]: This line is unnecessary

File: css-four-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  width: 100%;
+  margin: auto;
+}
+
+.icon {
+  position: relative;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:39]: This line is unnecessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Seungjoo! A couple minor notes here:

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**